arviz/plots/backends/matplotlib/khatplot.py

```diff
@@ -93,6 +93,7 @@ def plot_khat(
                 rgba_c = to_rgba_array(np.full(n_data_points, color))
         else:
             legend = False
+            color = (color - color.min()) / (color.max() - color.min())
```

Comment:
This has to come after the except. At this point `color` can be a list of strings or a nx3 array of rgb colors. I would also suggest using matplotlib normalizer function in case someone wanted to use a different one

```
cmap_name = kwargs.get("cmap", plt.rcParams["image.cmap"])
cmap = getattr(cm, cmap_name)
norm_fun = kwargs.get("norm", mpl.colors.Normalize(color.min(), color.max()))
rgba_c = cmap(norm_fun(color))
```